SCClassLibrary/Common/GUI/PlusGUI/Math/PlotView.sc

```diff
@@ -977,6 +975,30 @@ Plotter {
 		};
 	}
 
+	// Convenience methods to dispatch settings shared between all plot grids
```

Comment:
Would like feedback on this implementation:
These are convenience methods to set tick label properties of all of the subplots. 
In order to make the "setters", they also had to be defined as instance variables of `Plotter`. 
Because there are 2 axes with ticks (`x` and `y`), rather than making twice as many methods, I made the axis a variable. As a result the "getters" take an `ax` argument. Getters that take arguments seems odd, but perhaps ok in this scenario?
The getters